src/Microsoft.ML.Data/Transforms/ValueToKeyMappingEstimator.cs

```diff
@@ -22,13 +25,16 @@ namespace Microsoft.ML.Transforms
     /// | Input column data type | Scalar or vector of numeric, boolean, [text](xref:Microsoft.ML.Data.TextDataViewType), [System.DateTime](xref:System.DateTime) and [key](xref:Microsoft.ML.Data.KeyDataViewType) type. |
     /// | Output column data type | Scalar or vector of [key](xref:Microsoft.ML.Data.KeyDataViewType) type. |
     ///
-    /// The ValueToKeyMappingEstimator builds up term vocabularies(dictionaries) mapping the input values to the keys on the dictionary.
-    /// If multiple columns are used, each column builds/uses exactly one vocabulary.
-    /// The output columns are KeyDataViewType-valued.
-    /// The Key value is the one-based index of the item in the dictionary.
-    /// If the key is not found in the dictionary, it is assigned the missing value indicator.
-    /// This dictionary mapping values to keys is most commonly learnt from the unique values in input data,
-    /// but can be defined through other means: either with the mapping defined, or as loaded from an external file.
+    /// The ValueToKeyMappingEstimator maps the input values to keys using a
+    /// dictionary that is built during training. The dictionary mapping values to
+    /// keys is most commonly learnt from the unique values in input data,
```

Comment:
>most commonly [](start = 16, length = 13)

is this needed?

Reply:
Do you mean the whole sentence, or the word "most"?